### PR TITLE
Edit todo

### PR DESCRIPTION
**Pull request by StarMindz**
I added features to allow for editing and removing to-do list items

### CODE REVIEW

File: src/modules/tasks.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      const parent = e.target.parentNode;
+      const mainParent = parent.parentNode;
+      const id = e.target.id.split('_')[0];
+      const localcount = id - this.delcounter;
+      tasks.removeChild(mainParent);
+      this.taskList.splice(localcount, 1);
+      this.delcounter += 1;
+      this.dataLog();
+    });
+  }
```
Comment (Reem-lab) [inline src/modules/tasks.js]: - [ ] Great working on  the functionality for the removing but the indexes still the not right if i removed from the middle the indexex still not ordered so make it ordered as required 🚀 Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi @StarMindz, 👋🏻👋🏻

Good job so far! 🔥🔥🚀🚀
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights ✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality ✅
📌 Good commit messages ✅
📌Professional README file. ✅



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by StarMindz** [comment]
@Reem-lab Thank you for your review🥂.  I've made the necessary changes

**Review by Lordkaito — APPROVED** [review]
Hi @StarMindz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**